### DISCUSSION TITLE
Scythe Anticheat v2.19.0

### DISCUSSION THREAD

**Discussion by MrDiamond64** [Scythe-Anticheat/Scythe-Anticheat#121] [Announcements]
Scythe Anticheat is now at version 2.18.0!

Supported Minecraft versions: 1.20.50, 1.20.51

# Changelog
- Renamed Tower/A to Scaffold/A
- Renamed Aim/A to Scaffold/B
- Converted the worldborder module to use the Scripting API
- Converted !resetwarns command to use the Scripting API
- Enabled Namespoof/A and Namespoof/B to be enabled by default
- Fixed Scripting API checks not properly sending alerts to players with the `notify` tag
- Bug fixes
- Performance optimizations

By downloading this addon, you agree to our [license](https://github.com/Scythe-Anticheat/Scythe-AntiCheat/blob/main/LICENSE)
Questions? Join our [discord](https://discord.gg/pKyu95mYCT)!

<hr /><em>This discussion was created from the release <a href='https://github.com/Scythe-Anticheat/Scythe-AntiCheat/releases/tag/v2.19.0'>Scythe Anticheat v2.19.0</a>.</em>

**Comment by not-andy-ce** [comment]
Hey @MrDiamond64 why am I banned from the discord server? Can I be unbanned? Idfk what it's problem is

**Reply by MrDiamond64** [reply]
> Hey @MrDiamond64 why am I banned from the discord server? Can I be unbanned? Idfk what it's problem is

What is your discord username? I can check logs and see why you are banned.

**Reply by not-andy-ce** [reply]
> > Hey @MrDiamond64 why am I banned from the discord server? Can I be unbanned? Idfk what it's problem is
> 
> What is your discord username? I can check logs and see why you are banned.

It's thechosenoneyt0

**Reply by MrDiamond64** [reply]
> > > Hey @MrDiamond64 why am I banned from the discord server? Can I be unbanned? Idfk what it's problem is
> > 
> > What is your discord username? I can check logs and see why you are banned.
> 
> It's thechosenoneyt0

We've had a little encounter at January 22, 2023 regarding you posting defamatory content in an attempt to ruin the Scythe name. You will not be unbanned.

**Reply by not-andy-ce** [reply]
> > > > Hey @MrDiamond64 why am I banned from the discord server? Can I be unbanned? Idfk what it's problem is
> > > 
> > > What is your discord username? I can check logs and see why you are banned.
> > 
> > It's thechosenoneyt0
> 
> We've had a little encounter at January 22, 2023 regarding you posting defamatory content in an attempt to ruin the Scythe name. You will not be unbanned.

I barely remember that. Is there a way I can make up for it? All I want is to join the server...
